src/cpp/src/make_tokenizer_stateful.cpp

```diff
@@ -14,21 +14,12 @@ using namespace ov;
 using namespace ov::op;
 
 bool ov::genai::MakeCombineSegmentsSatateful::run_on_model(const std::shared_ptr<ov::Model>& model) {
-
     std::shared_ptr<ov::Node> combine_seg_node;
     for (auto node: model->get_ordered_ops()) {
         if (strcmp(node->get_type_info().name, "CombineSegments") == 0) {
             combine_seg_node = node;
         }
     }
-    if (!combine_seg_node || combine_seg_node->input_value(1).get_element_type() != ov::element::i32) {
```

Comment:
Because of the code above it's always not null. Moreover input type for `CombineSegments` is always i32, this is ensured in evaluate_and_infer_types().